superset-frontend/src/dashboard/stylesheets/components/chart.less

```diff
@@ -62,6 +62,7 @@
 .dashboard-chart {
   overflow: hidden;
   position: relative;
+  flex-shrink: 0;
```

Comment:
I'm a bit concerned by this change as it could impact other charts (but I have no idea what this does) and it's in the Superset code base.

Reply:
yea im not the biggest fan of base superset code modifications. However if we do need it we should also try to put in an upstream PR